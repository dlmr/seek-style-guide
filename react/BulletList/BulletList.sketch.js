import React from 'react';
import { BulletList, Bullet } from 'seek-style-guide/react';
export const symbols = {
  'BulletPoint/Small': (
    <BulletList>
      <Bullet small>Bullet point 1</Bullet>
      <Bullet small>Bullet point 2</Bullet>
      <Bullet small>Bullet point 3</Bullet>
    </BulletList>
  ),
  'BulletPoint/Standard': (
    <BulletList>
      <Bullet>Bullet point 1</Bullet>
      <Bullet>Bullet point 2</Bullet>
      <Bullet>Bullet point 3</Bullet>
    </BulletList>
  ),
  'BulletPoint/Subheading': (
    <BulletList>
      <Bullet subheading>Bullet point 1</Bullet>
      <Bullet subheading>Bullet point 2</Bullet>
      <Bullet subheading>Bullet point 3</Bullet>
    </BulletList>
  )
};
