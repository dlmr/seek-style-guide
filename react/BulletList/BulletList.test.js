import React from 'react';
import { shallow } from 'enzyme';

import Text from '../Text/Text';

import BulletList from './BulletList';

describe('BulletList', () => {
  it('renders with standard text', () => {
    const wrapper = shallow(
      <BulletList>
        <Text>Foo</Text>
        <Text>Bar</Text>
      </BulletList>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
