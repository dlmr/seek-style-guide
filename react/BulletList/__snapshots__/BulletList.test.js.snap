// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BulletList renders with standard text 1`] = `
<ul
  className="root"
>
  <Text>
    Foo
  </Text>
  <Text>
    Bar
  </Text>
</ul>
`;
